### PR TITLE
Updated model-viewer, puppeteer, and other dependencies

### PR DESCRIPTION
**Pull request by sleroux**
This PR upgrades all the dependencies including <model-viewer> to the latest version 1.6.0. There seemed to be a problem with another PR that resulted in the models being rendered to the top left corner but after testing this with different models in the 3D Model Service I can't seem to reproduce that issue.

Note that there were some extra spaces at the end of some lines that were included in this PR that were removed. No code changes were made.

### CODE REVIEW

**Review by tylerrowsell — APPROVED** [review]

**Comment by sleroux** [comment]
I'll write a quick script to run through models over and over locally to see if it takes a high amount of volume to reproduce the issue. If nothing comes of that, let's ship this alongside the old one but don't use the new version yet. Instead, we can run some image diff analysis between the new and old snapshots behind the scenes and ping Bugsnag if there is a large difference.

**Comment by sleroux** [comment]
Ah ha! I found a way to replicate this issue. I setup a docker container and ran it at 70% CPU capacity and noticed that some of the screenshots were skewing to the top left and were incorrect. Definitely something with how much time it takes for slower/VM'ed machines to render.

File: src/capture-screenshot.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+          background-color=""
+          id="snapshot-viewer"
+          interaction-prompt="none"
+          src="${inputPath}" />
+      </body>
+    </html>
+  `
+}
+
+const captureScreenshot = async function (options) {
```
Comment (sleroux) [inline src/capture-screenshot.js:43]: Most of this code is tweaked and borrowed from how <model-viewer> does their fidelity testing here: https://github.com/google/model-viewer/blob/master/packages/render-fidelity-tools/src/artifact-creator.ts#L253. I could have just taken the relevant `poster-visibility`hack they implemented into our existing flow but to be honest this was much cleaner with less timeout logic.

File: src/capture-screenshot.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+      let timeout;
+      if (maxTimeInSec > 0) {
+        timeout = setTimeout(() => {
+          reject(new Error(
+              `Stop capturing screenshot after ${maxTimeInSec} seconds`));
+        }, maxTimeInSec * 1000);
+      }
+
+      const modelViewer = document.getElementById('snapshot-viewer');
+      modelViewer.addEventListener('poster-dismissed', () => {
```
Comment (sleroux) [inline src/capture-screenshot.js:100]: This is the 'hack' model-viewer does to help with the timing of screenshoting a rendered model: https://github.com/google/model-viewer/blob/master/packages/render-fidelity-tools/test/renderers/model-viewer/index.html#L49. Turns out it works pretty well! I ran a suite of models on a slower VM using Docker and managed to get the images rendering correctly where as before they were all skewed to the top left corner or weird sizes.

File: src/capture-screenshot.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+          model-viewer {
+            --progress-bar-color: transparent;
+            width: ${width};
+            height: ${height};
+          }
+        </style>
+      </head>
+      <body>
+        <model-viewer
+          style="background-color: ${backgroundColor};"
```
Comment (sleroux) [inline src/capture-screenshot.js:33]: Ported over our support for setting a background color

**Review by tylerrowsell — APPROVED** [review]
Changes worked for me! 🎉

File: src/capture-screenshot.js [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+          id="snapshot-viewer"
+          interaction-prompt="none"
+          src="${inputPath}" />
+      </body>
+    </html>
+  `
+}
+
+const captureScreenshot = async function (options) {
+  const browserT0 = performance.now();
```
Comment (mikkoh) [inline src/capture-screenshot.js:44]: These could probably be cleaned up by doing something like:
`performance.mark('browserStartup');`
Then later:
`performance.measure('browserStartup').duration`

Comment (sleroux) [reply src/capture-screenshot.js:44]: They ended up removing returning the PerformanceEntry from the return of measure in Node 10 and prefer to use PerformanceObserver which makes the code a bit more involved.

**Pull request merged**